home.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Flawless Shop</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-icons">
+        <ul>
+         <li><i class="fa fa-twitter"></i></li> 
+         <li><i class="fa fa-instagram"></i></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png" alt="logo"></a>
+        <div class="nav-links">
+          <a class="LINK" href="about.html">About</a>
+          <a class="LINK" href="#products">Our Products</a>
+          <a class="LINK"href="#Artists">Artists</a>
+          <a href="" class="flawless">FLAWLESS</a>
+        </div>
+      </div>
+      <i class="fa fa-bars fa-2x" id="bars"></i>
+    </nav>
+    <div class="nav-menu">
+      <i class="fa fa-times fa-2x" id="closingIcon"></i>
+      <ul>
+        <li class="mobileLinks"><a href="home.html">Home</a></li>
+        <li class="mobileLinks"><a href="#products">Products</a></li>
+        <li class="mobileLinks"><a href="about.html">About</a></li>
+      </ul>
+    </div>
+    <div class="main">
+      <q>Hi there!</q>
+      <h1>With Flawless, Glow like never before!</h1>
+      <p class="description">
+        Looking young and beautiful never comes without using the best cosmetics.
+        Here we have compiled the best skincare products by the best brands.
+      </p>
+      <p class="promotion">
+        10% OFF FOR FIRST PURCHASE
+      </p>
```

Comment:
- [ ] I have noticed that the description and promotion overlaps into the items below at screen size of around 900px. This makes the application less user-friendly 😞 . This is caused mainly by the `height` on the header, kindly consider removing the height and apply the styles to match the original design as much as possible.☺️ 

Your design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/6aa030d3-5fa6-4e5d-9f29-1bf3371c9246)

Original design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/eca36e5f-e7aa-417f-bf25-d9208c9f3cfe)